File: Basic Java/TestingSystem_Assignment/src/com/vti/backend/TestingSystem_Assignment_4/NhanVien.java
package com.vti.backend.TestingSystem_Assignment_4;

public class NhanVien extends CanBo{
    private String job;

    public NhanVien(long id, String fullName, int age, Gender gender, String address, String job) {
        super(id, fullName, age, gender, address);
        this.job = job;
    }
}

File: ReactJS/Assignment_02/Lab_02/NhanVien.ts
import {CanBo, Gender} from './CanBo';


export class NhanVien extends CanBo {
    private job: string;


	constructor(fullName: string, age: number, gender: Gender, address: string, job: string) {
        super(fullName, age, gender, address);
		this.job = job;
	}



    /**
     * Getter job
     * @return {string}
     */
	public getJob(): string {
		return this.job;
	}

    /**
     * Setter job
     * @param {string} value
     */
	public setJob(value: string) {
		this.job = value;
	}


    public displayDetails(){
        super.displayDetails();
        console.log( `Job: ${this.getJob()} `);
    }
    
}
